src/ui/components/FocusContextReduxAdapter.tsx

```diff
@@ -2,31 +2,32 @@ import { TimeStampedPointRange } from "@replayio/protocol";
 import { PropsWithChildren, useCallback, useDeferredValue, useMemo } from "react";
 
 import { FocusContext, UpdateOptions } from "replay-next/src/contexts/FocusContext";
+import { useCurrentFocusWindow } from "replay-next/src/hooks/useCurrentFocusWindow";
 import { TimeRange } from "replay-next/src/types";
 import {
   enterFocusMode,
-  setFocusWindow,
-  setFocusWindowImprecise,
+  setFocusWindow as setDisplayedRange,
+  setFocusWindowImprecise as setDisplayedRangeImprecise,
   syncFocusedRegion,
   updateFocusWindowParam,
 } from "ui/actions/timeline";
 import { getFocusWindow } from "ui/reducers/timeline";
 import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
-import { rangeForFocusWindow } from "ui/utils/timeline";
 
 // Adapter that reads focus region (from Redux) and passes it to the FocusContext.
 export default function FocusContextReduxAdapter({ children }: PropsWithChildren) {
   const dispatch = useAppDispatch();
-  const focusWindow = useAppSelector(getFocusWindow);
+  const range = useCurrentFocusWindow();
+  const rangeForDisplay = useAppSelector(getFocusWindow);
```

Comment:
I think it would be nice to remove focus range from Redux entirely so we could avoid hoops like this. (Not something we need to do _now_, just thinking out loud.)